### PR TITLE
Add configuration documentation

### PR DESCRIPTION
**Pull request by aledeg**
Writting documentation is really long. Writting good documentation is really hard.
I hope the changes I've made are worth it.

### CODE REVIEW

File: docs/en/users/05_Configuration.md [hunk 1]
```diff
@@ -3,9 +3,29 @@
...
 ## Language
 
-At the moment, FreshRSS is available in French and English. After you confirm your choice, the whole interface will be displayed in the chosen language.
-
-There are parts of FreshRSS that are not translated and are not intended to be translated. For now, the logs visible in the application as well as the one generated by the script of automatic update are part of it.
+At the moment, FreshRSS is available in 13 languages. After you confirm your choice, the interface will be displayed in the chosen language.
+Depending on the chosen language, there might be parts of the interface that are still not translated. If you're willing to help translating
+the missing bits or add a new language, please check how you can [contribute to the project](../contributing.md#contribute-to-internationalization-i18n).
+
+There are parts of FreshRSS that are not translated and are not intended to be translated. For now, the logs visible in the application as well as the one generated by automatic update script are part of it.
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: the automatic update script (by automatic update script*s*?)

File: docs/en/users/05_Configuration.md [hunk 2]
```diff
@@ -3,9 +3,29 @@
...
+
+Not all languages are equals regarding completion, here is the completion status:
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: Comma splice, but we can probably just drop the second part of the sentence.

File: docs/en/users/05_Configuration.md [hunk 3]
```diff
@@ -33,14 +53,31 @@ There are some who prefer short lines of text while others prefer to maximize th
 
 ## Article icons
 
-**TODO**
+It worth noting that this section have effects only in normal view.
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: It's worth nothing

this section (singular) only has an effect in normal view

File: docs/en/users/05_Configuration.md [hunk 4]
```diff
@@ -51,15 +88,53 @@ The duration of this notification can be set. By default, the value is 0.
...
+To remove an item from the list, follow those simple steps:
+
+ 1. Press the ```❌``` sign next to the share method you want to remove.
+ 1. Submit your changes.
 
 # Shortcuts
 
-**TODO**
+To ease the use of the application, FreshRSS comes with a lot of predefined keyboard shortcuts.
+They allow a lot of actions to improve the user experience with a keyboard.
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: Maybe `They offer many actions`? (`comes with a lot` immediately followed by `allow for a lot`)

File: docs/en/users/05_Configuration.md [hunk 5]
```diff
@@ -51,15 +88,53 @@ The duration of this notification can be set. By default, the value is 0.
...
+Of course, if you're not satisfied with the key mapping, you can change you configuration to fit your needs.
+
+There are 4 types of shortcuts:
+
+ 1. Views: they allow switching views with ease.
+ 1. Navigation: they allow navigation through articles, feeds, and categories.
+ 1. Article actions: they allow interactions with an article, like sharing or opening it on the original web-site.
+ 1. Other actions: they allow other interactions with the application, like opening the user queries menu or accessing the documentation.
+
+It's worth noting that the share article action has two levels. Once you pressed the shortcut, a menu containing all the share options opens.
```
Comment (Frenzie) [inline docs/en/users/05_Configuration.md]: press

**Comment by Frenzie** [comment]
I think they are! (As long as people read the manual. :-P)

**Comment by aledeg** [comment]
@Frenzie I've change the doc. Thank you for proof-reading it.

**Comment by Alkarex** [comment]
Nice improvements 👍

**Pull request merged**